src/LaunchDarkly.OpenFeature.ServerProvider/Provider.StatusProvider.cs

```diff
@@ -11,6 +11,7 @@ public sealed partial class Provider
         private class StatusProvider
         {
             private ProviderStatus _providerStatus = ProviderStatus.NotReady;
```

Comment:
This is now a mirror of the status because the provider doesn't own its status, but also cannot read it.